source/collection-management/tests/timeseries-collection.yml

```diff
@@ -0,0 +1,59 @@
+description: "timeseries-collection"
+
+schemaVersion: "1.0"
+
+runOnRequirements:
+  - minServerVersion: "5.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name ts-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name test
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents: []
+
+tests:
+  - description: "createCollection with all options"
+    operations:
+      - name: dropCollection
+        object: *database0
+        arguments:
+          collection: *collection0Name
+      - name: createCollection
+        object: *database0
+        arguments:
+          collection: *collection0Name
+          timeseries: &timeseries0
+            timeField: "time"
+            metaField: "meta"
+            expireAfterSeconds: 604800
+      - name: assertCollectionExists
+        object: testRunner
+        arguments:
+          databaseName: *database0Name
+          collectionName: *collection0Name
+    expectEvents:
+      - client: *client0
+        events:
+          - commandStartedEvent:
+              command:
+                drop: *collection0Name
+              commandName: drop
```

Comment:
FWIW, I don't think `commandName` is very useful for APM assertions. I'd be in favor of dropping this, as most of our other tests don't use it. I actually don't recall why it was originally introduced in the older test formats, and I neglected to answer that question when pulling it forward into the unified format.

That said, this _is_ relevant for the `runCommand` operation (same option name, entirely different context) to overcome JSON/YAML key order concerns.

Reply:
Ah interesting; seems like it `commandName` serves no purpose here. Removed the assertions.